BackofficeBundle/Resources/public/coffee/FullPageFormView.coffee

```diff
@@ -2,21 +2,21 @@ FullPageFormView = OrchestraView.extend(
   el: '#content'
 
   initialize: (options) ->
+    @initializer options
+    @loadTemplates [
+      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/fullPagePanelView',
+      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/elementTitle',
+    ]
+    return
+
+  initializer: (options) ->
     @options = options
     @options.listUrl = appRouter.generateUrl('listEntities', entityType: options.entityType) if options.listUrl == undefined
     @completeOptions(@options.element) if @options.element != undefined
     @events = @events || {}
-    @loadTemplates [
-      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/fullPageFormView',
-      'OpenOrchestraBackofficeBundle:BackOffice:Underscore/elementTitle'
-    ]
-    return
 
   render: ->
-    if @options.multiVersion
-      @options.title = @renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/elementTitle',
-        element: @options.element
-      )
+    @callMultiVersionOptions()
     $(@el).html(@renderTemplate('OpenOrchestraBackofficeBundle:BackOffice:Underscore/fullPageFormView', @options))
```

Comment:
On my mind, this won't work as the template is no more loaded on the initialization